internal/boxcli/list.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Jetify Inc. and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package boxcli
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"go.jetpack.io/devbox/internal/devbox"
+	"go.jetpack.io/devbox/internal/devbox/devopt"
+)
+
+type listCmdFlags struct {
+	config configFlags
+}
+
+func listCmd() *cobra.Command {
+	flags := listCmdFlags{}
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List the packages in your current devbox project",
```

Comment:
Since this is also used for global, maybe a more generic:

`List installed packages`

Reply:
Updated!